src/integration/resources/inprocess/coordinator.go

```diff
@@ -248,6 +254,12 @@ func (c *coordinator) WaitForShardsReady() error {
 }
 
 func (c *coordinator) Close() error {
+	if c.embedded {
```

Comment:
Would be nice if this just handled double closes sanely regardless of embedded or not.